closed/get_openssl_source.sh

```diff
@@ -72,7 +72,7 @@ case "$OPENSSL_VERSION" in
 		OPENSSL_SOURCE_TAG="openssl-$OPENSSL_VERSION"
 		;;
 	*)
-		usage
+		OPENSSL_SOURCE_TAG=$OPENSSL_VERSION
```

Comment:
While reviewing this, I notice that this script will fail if `OPENSSL_VERSION` includes spaces. I'm not sure whether or not we should be concerned about that. Opinion(s)?

Reply:
I wasn't planning to use any space in a branch or tag name.

Reply:
I don't think git allows branch or tag names to have spaces, so you would have no need to specify `OPENSSL_VERSION` to be a value with spaces, but I suppose my question was whether the script should only accept values that are reasonable.

Reply:
We've managed this long without it, it doesn't seem important.

Reply:
Agreed, but I thought it better the assumption be mentioned.